node/src/Cardano/Node/API.hs

```diff
@@ -259,10 +259,11 @@ getNodeSettings compileInfo updateConfiguration timestamp slottingVar ns = do
 
     slotId            <- liftIO $ getTipSlotId ns
     maxTxSize         <- liftIO $ getMaxTxSize ns
-    feePolicy         <- liftIO $ getFeePolicy ns
     securityParameter <- liftIO $ getSecurityParameter ns
     slotCount         <- liftIO $ getSlotCount ns
-
+    feePolicy         <- liftIO $ getFeePolicy ns >>= \case
+        Core.TxFeePolicyTxSizeLinear a -> return a
+        _ -> fail "getNodeSettings: Unsupported / Unknown fee policy."
```

Comment:
And this is where we glue the type alias for `feePolicy`. Quite simple, but it makes our life a lot easier at the API-level.